openapi.yaml

```diff
@@ -469,1250 +491,1393 @@ paths:
+    '409':
+      description: Type or ID conflict
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+    '500':
+      description: Internal server error
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+  securitySchemes:
+    OAuth2:
+      type: oauth2
+      flows:
+        clientCredentials:
+          tokenUrl: https://api.oregonstate.edu/oauth2/token
+          scopes:
+            full: Full access to the API
+  schemas:
+    PersonType:
+      type: string
+      enum: [person]
+    JobType:
+      type: string
+      enum: [job]
+    SsnType:
+      type: string
+      enum: [ssn]
+    MealPlanType:
+      type: string
+      enum: [meal-plans]
+    AddressType:
+      type: string
+      enum: [addresses]
+    PhoneType:
+      type: string
+      enum: [phones]
+    EmailType:
+      type: string
+      enum: [emails]
+    PersonPostRequired:
+      type: object
+      required:
+        - firstName
+        - lastName
+        - birthDate
+        - sex
+    PersonPostAttributes:
+      type: object
+      additionalProperties: false
+      properties:
+        firstName:
+          type: string
+          description: First name of person
+          example: John
+        lastName:
+          type: string
+          description: Last name of person
+          example: Smith
+        middleName:
+          type: string
+          nullable: true
+          description: Middle name of person
+          example: Adam
+        birthDate:
+          type: string
+          format: date
+          description: Birthday of person
+          example: '1970-01-21'
+        citizen:
+          type: object
           properties:
-            effectiveDate:
+            code:
               type: string
-              format: date
-              description: Date this labor distribution becomes effective.
-              example: '1970-01-01'
-            accountIndex:
-              type: object
-              properties:
-                code:
-                  type: string
-                  example: ABC123
-            fund:
-              type: object
-              properties:
-                code:
-                  type: string
-            organization:
+              nullable: true
+              enum: [FN, N, R, S, C]
+              description: |
+                Person's citizen code
+                - `FN` - Foreign National
+                - `N` - Non Resident Alien
+                - `R` - Resident Alien
+                - `S` - Substantial Presence Alien
+                - `C` - Citizen
+        sex:
+          type: string
+          nullable: true
+          enum:
+            [M, F, N]
+          description: |
+            Person's sex code
+            - `M` - Male
+            - `F` - Female
+            - `N` - Non-specified
+    PersonAttributes:
```

Comment:
I don't think any of these attributes can be null beside `displayMiddleName` for a good person data. Since we have `nullable` in openapi 3, it's handy for us to check if the return data is expected or not.